client/test/Test/Pos/Client/Txp/UtilSpec.hs

```diff
@@ -286,19 +331,21 @@ makeManyUtxoTo1Params numFrom amountEachFrom amountTo = do
     pure CreateMTxParams {..}
 
 makeManyAddressesToManyParams
-    :: Int
+    :: InputSelectionPolicy
+    -> Int
     -> Integer
     -> Int
     -> Integer
     -> Gen CreateMTxParams
-makeManyAddressesToManyParams numFrom amountEachFrom numTo amountEachTo = do
+makeManyAddressesToManyParams inputSelectionPolicy numFrom amountEachFrom numTo amountEachTo = do
```

Comment:
Since `Gen` appeared here I think it would be nice to take arbitrary `InputSelectionPolicy` where possible
(but this PR seems inappropriate to do such things)